hotsos/defs/scenarios/openstack/neutron/bugs/lp1996594.yaml

```diff
@@ -8,7 +8,7 @@ checks:
     input:
       path: 'var/log/neutron/neutron-ovn-metadata-agent.log'
     search:
-      expr: '([0-9-]+)\s([0-9:]+)\.\d{3}.+OVSDB Error: no error details.+'
+      expr: '([\d-]+) [\d:]+\.\d{3} .+OVSDB Error: no error details.+'
```

Comment:
Need to keep the grouping which can be one for whole timestamp or two for date and time e.g.

expr: '([\d-]+ [\d:]+\.\d{3}) .+OVSDB Error: no error details.+'